survey.html

```diff
@@ -1,12 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>survey-form</title>
+    <title>survey form</title>
 </head>
+
 <body>
-    <h1>hello people</h1>
+    <div id="title">
+        <h1>GeeksforGeeks Survey Form</h1>
+    </div>
+    <form id="form">
```

Comment:
I used form tag because is what is more suitable to the forms compared to the other HTML tags